OpenSim/Simulation/Model/BushingForce.h

```diff
@@ -80,15 +75,28 @@ OpenSim_DECLARE_CONCRETE_OBJECT(BushingForce, Force);
 //==============================================================================
 // PUBLIC METHODS
 //==============================================================================
-    /** Default constructor leaves bodies unspecified, sets the bushing frames
+    /** Default constructor leaves frames unspecified, sets the bushing frames
     to be at their body origins, and sets all bushing parameters to zero. **/
     BushingForce();
-    /** This constructor provides everything needed to create a real 
-    bushing. See property declarations for more information. **/
-    BushingForce(const std::string& body1Name, 
+
+    /** Construct the BushingForce given the offset frames that it tries to
+    align according to the physical properties of the bushing.
+    See property declarations for more information. **/
+    BushingForce(const PhysicalOffsetFrame& frame1,
```

Comment:
> The alternative is to assume the offset exists somewhere (e.g. in the Model), but this does not map well to the use cases/examples. 

I would have expected to be able to put a bushing (or joint or constraint) between any two PhysicalFrames. I don't see why it has to be limited to a PhysicalOffsetFrame. (Making one with a zero offset seems like a hack!) In some cases the PhysicalFrame would be a specially-built frame just for use with the bushing (say), but in others it would be a pre-existing frame. For example, I may have reason to attach one end of a bushing to the same frame as I'm using for one end of a joint.

I'm wondering if this problem has to do with *ownership* of the PhysicalFrame -- in some cases we would like to use an existing frame, in others we might want to craft special frames that "go with" the bushing. I think a bushing should connect to two PhysicalFrames. As an orthogonal issue, some component will be the owner of those frames; that component might be the bushing or might be someone else.